clients/cobol-lsp-vscode-extension/src/__tests__/services/ProcessorGroups.spec.ts

```diff
@@ -25,102 +26,80 @@ import { globSync } from "glob";
 
 const WORKSPACE_URI = "file:///my/workspace";
 
+const platform = process.platform;
+beforeAll(() =>
+  Object.defineProperty(process, "platform", {
+    value: "darwin",
+  }),
+);
+afterAll(() =>
+  Object.defineProperty(process, "platform", {
+    value: platform,
+  }),
+);
```

Comment:
What part of the code needs this?

Reply:
UriMock as Uri handles paths->fsPath conversion differently. At least I understand it so https://github.com/microsoft/vscode-uri/blob/6dec22d7dcc6c63c30343d3a8d56050d0078cb6a/src/uri.ts#L564

So to pass tests on windows machine, im set platform to non windows.

Reply:
I guess I can avoid it by checking for column

Reply:
I think that the root cause of this whole problem is that we have 2 versions of the test - one for windows, one for the rest, but we are always running both.
The only difference between those tests are 2 absolute paths. I suggest merging the two files to one.